setup.py

```diff
@@ -104,7 +104,7 @@ def spirv_compile():
     if IS_LIN:
         compiler = "icx"
     if IS_WIN:
-        compiler = "icx.exe"
+        compiler = "clang.exe"
```

Comment:
With `icx` it fallback to `clang-cl` which does not understand all our input arguments.